django/thunderstore/schema/sync.py

```diff
@@ -0,0 +1,84 @@
+import requests
+from django.db import transaction
+
+from thunderstore.community.models import (
+    Community,
+    PackageCategory,
+    PackageListingSection,
+)
+from thunderstore.core.settings import ECOSYSTEM_SCHEMA_ENDPOINT
+from thunderstore.schema.schema import Schema, SchemaGame
+
+
+# TODO: Add support for deleting or at least disabling unnecessary content
+@transaction.atomic
+def import_game_community(identifier: str, game: SchemaGame):
+    if not game.thunderstore:
+        return
+
+    config = game.thunderstore
+
+    if not (community := Community.objects.filter(identifier=identifier).first()):
+        community = Community(
+            identifier=identifier,
+            is_listed=False,
+            block_auto_updates=False,
+        )
+
+    if community.block_auto_updates:
+        return
+
+    community.slogan = f"The {game.meta.displayName} Mod Database"
```

Comment:
Is it intentional the slogan and description are always overwritten by hardcoded values? Should this be done on L22 instead when a new community is created?

Reply:
Really these fields should be removed from the Community model entirely as all of them follow this same pattern currently. As such I figured it's better to force them to remain in line with that pattern than it is to allow overwrites to it in the admin.